eureka-client/src/main/java/com/netflix/discovery/DiscoveryManager.java

```diff
@@ -42,6 +42,7 @@
 public class DiscoveryManager {
     private static final Logger logger = LoggerFactory.getLogger(DiscoveryManager.class);
     private DiscoveryClient discoveryClient;
+    private EurekaClient eurekaClient; // Might be a proxy to DiscoveryClient
```

Comment:
Can you call it something like `clientOverride`?

Reply:
Done